client/src/pages/NewReport/ApplicationSection/ApplicationSection.tsx

```diff
@@ -0,0 +1,151 @@
+import SectionComponent from 'components/SectionComponent/SectionComponent.tsx';
+import Table, { TableColumn } from 'components/Table/Table.tsx';
+import { useEffect, useState } from 'react';
+import TagButton from 'components/TagButton/TagButton.tsx';
+import SVGIcon from 'components/SVGIcon/SVGIcon.tsx';
+import ActionButton, { ActionButtonColor } from 'components/ActionButton/ActionButton.tsx';
+import OverlayComponent from 'components/OverlayComponent/OverlayComponent.tsx';
+import LinkComponent from 'components/LinkComponent/LinkComponent.tsx';
+import CustomPrompt from 'components/CustomPrompt/CustomPrompt.tsx';
+import { ManagmentServiceApiInstance } from 'api/managment-service';
+
+interface ApplicationDataRow {
+    id: string;
+    name: string;
+    accuracy: 'HIGH' | 'MEDIUM' | 'LOW';
+    customPrompt: string;
+    updated: string;
+    added: string;
```

Comment:
Are you sure we have an id of an application in the contract with management-service?

Reply:
yeah, you are right